### DISCUSSION TITLE
How to hide extraneous boot options on Clover GUI?

### DISCUSSION THREAD

**Discussion by ACDC77** [CloverHackyColor/CloverBootloader#581] [Q&A] [answered]
I have a working instance of Clover release 5150 on a Dell PC. I have the Clover volume on a SATA drive (instead of a USB) and Windows 10 on a NVMe SSD on a PCI adapter. It's working great except that I’m unable to hide extraneous boot options in the Clover GUI. I’ve followed suggestions on several forums but nothing seems to work. I posted a detailed write-up on everything I've tried on the [Win-Raid Forum](https://winraid.level1techs.com/t/guide-nvme-boot-without-modding-your-uefi-bios-clover-efi-bootloader-method/31665/861). Any help would be appreciated.

Would it be OK to repost the detailed write-up on this forum? I'm not sure of the protocol. Therefore, I've posted a link to my detailed write-up for now.

**Comment by ACDC77** [comment, marked as answer]
Just following up here to say that I solved my problem. The issue was my ignorance of the plist file format. I believe the question mark at the end of a key in the default config.plist simply indicates that there is zero of one occurrence of the key. One must include the “Hide” key without the trailing question mark and the desired array string values for this to work. The same result can also be achieved by using the "Scan" key and setting "Legacy" as "false" as indicated in [my latest post at the Win-Raid forum](https://winraid.level1techs.com/t/guide-nvme-boot-without-modding-your-uefi-bios-clover-efi-bootloader-method/31665/862).

**Reply by spsharma2018** [reply]
I have been using Clover for almost 10+ years and had a successful dual boot system with MacOS + Win 10. In old EFI folders, I had three sub-folders Apple, Clover and Microsoft. For dual boot, I would crate a custom boot in "GUI" with a path "/EFI/Microsoft/Boot/bootmgfw.efi " pointing to the Microsoft folder in the win disk EFI volume (GPT). Now-a-days, I can not find the Microsoft folder. My Microsoft folder are very old. I have a Window boot item with "boot from Windows" with no UUID to address to. I do not know howto hide it, since it is useless.
